metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -740,22 +735,24 @@ public Long apply(List<ApiMessageAndVersion> records) {
                             // Start by trying to apply the record to our in-memory state. This should always
                             // succeed; if it does not, that's a fatal error. It is important to do this before
                             // scheduling the record for Raft replication.
-                            int i = 1;
+                            int i = 0;
```

Comment:
I realize it already existed, but can we rename `i` to something more descriptive